_posts/2013-05-02-install.markdown

```diff
@@ -249,9 +250,14 @@ gem -v
 
 ### *2.* התקיני עורך טקסט לעריכת קבצי האפליקציה
 
-אנחנו ממליצות להוריד את Sublime
+אנחנו ממליצים להוריד עורך טקסט בשם Atom לצורך הסדנא.
 
-[Sublime Text 2](http://www.sublimetext.com/2)
+* [הורידי Atom והתקיני אותו:](https://github.com/atom/atom/releases/latest)
+  * הורידי את הקובץ zip ל- windows ופתחי אותו.
+  * העתיקי את התיקייה לתוך Program Files ופתחי את Atom משם.
+
+אם את משתמשת ב Windows Vista או בגרסאות ישנות יותר, את יכולה להוריד את העורך של
```

Comment:
maybe it's better to move this up - so they will see it before doing the above steps